src/main/java/com/google/sps/classes/Utility.java

```diff
@@ -52,7 +52,7 @@ public static int getUserId() {
     String email = userService.getCurrentUser().getEmail();
 
     // If user is not logged in, return -1.
-    if (email = "") {
+    if (email == "") {
```

Comment:
I believe you should always use .equals for string comparison in Java - https://stackoverflow.com/questions/513832/how-do-i-compare-strings-in-java

Reply:
That's right! We just had an issue with that and I completely forgot! Fixed it here but will also make a hotfix about it, might be causing issues to the others.